rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -214,14 +214,18 @@ Apply Configuration
 
       kubectl edit secret -n flyte flyte-propeller-auth
 
-   Add a new key under `stringData`:
+   Create the `stringData` block if one does not already exist, and add the `client_secret` key under it, as follows:
```

Comment:
Thank you for updating this. This section deals with the custom Authorization Server setup. However, given that we are removing client_secret from the default secrets, we need to add a section for configuring the SelfAuthorizationServer as well. Otherwise that won't work OOB